### DISCUSSION TITLE
How to generate XML Report from cypress-allure-plugin

### DISCUSSION THREAD

**Discussion by Animeshkr29** [allure-framework/allure2#1318] [Questions & Support] [answered]
Hi Developers Team

i want to integrate test rail with Allure Cypress report , can any one please help to know how to get XML Report if i am using Allure Report Plugin in Cypress .

**Comment by Avaddon-tfa** [comment]
Hi! Allure uses adapters to gather test cases data. The first versions of the adapter used XML files, but it has been updated to JSON in the second version. We don't recommend using the deprecated adapter and XML format. 

Using a new adapter version with JSON is a way to go.

In case you are hard stuck on the XML way, please, describe the exact use case and we'll try to help.

**Reply by Animeshkr29** [reply, marked as answer]
Thanks for reply , 
i think my requirement is full filled with junit Report which generate XML result ,
